src/pages/my_registrations/MyRegistrationsList.tsx

```diff
@@ -66,105 +74,49 @@ export const MyRegistrationsList = ({ registrations, refetchRegistrations }: MyR
       setIsDeleting(false);
     } else if (isSuccessStatus(deleteRegistrationResponse.status)) {
       dispatch(setNotification({ message: t('feedback.success.delete_registration'), variant: 'success' }));
-      refetchRegistrations();
+      refetchRegistrations && refetchRegistrations();
```

Comment:
Denne er ikke optional?
```suggestion
      refetchRegistrations();
```